tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -1010,7 +1014,14 @@ def _cudnn_rnn(inputs,
       "seed2": seed2,
       "name": name
   }
-  if use_cudnn_v2 != "1":
+  if sequence_lengths is not None:
+    if sequence_lengths.dtype != dtypes.int32:
```

Comment:
No need for this check, for the OpDef in `ops/cudnn_rnn_ops.cc` will be used to catch dtype error during run time.